aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -375,36 +378,36 @@ To enforce HTTPS, install the `mod_rewrite` module to enable URL rewriting:
 sudo yum install mod_rewrite
 ```
 
-Modify the *helloapp.conf* file to enable URL rewriting and secure communication on port 443:
+Modify the *helloapp.conf* file to enable secure communication on port 443.
+
+The following example doesn't configure the server to redirect insecure requests. We recommend using HTTPS Redirection Middleware. For more information, see <xref:security/enforcing-ssl>.
+
+> [!NOTE]
+> For development environments where the server configuration handles secure redirection instead of HTTPS Redirection Middleware, we recommend using temporary redirects (302) rather than permanent redirects (301). Link caching can cause unstable behavior in development environments.
+
+Adding a `Strict-Transport-Security` (HSTS) header ensures all subsequent requests made by the client are over HTTPS. For guidance on setting the `Strict-Transport-Security` header, see <xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts>.
 
 ```
 <VirtualHost *:*>
     RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
 </VirtualHost>
 
-<VirtualHost *:80>
-    RewriteEngine On
-    RewriteCond   %{HTTPS} !=on
-    RewriteRule   ^(.*)$ https://%{HTTP_HOST}$1 [R=301,L]
-</VirtualHost>
-
 <VirtualHost *:443>
     Protocols             h2 http/1.1
     ProxyPreserveHost     On
     ProxyPass             / http://127.0.0.1:5000/
     ProxyPassReverse      / http://127.0.0.1:5000/
     ErrorLog              /var/log/httpd/helloapp-error.log
     CustomLog             /var/log/httpd/helloapp-access.log common
-    Header                always set Strict-Transport-Security "max-age=63072000;includeSubdomains;preload"
     SSLEngine             on
-    SSLProtocol           all -SSLv3 -TLSv1 -TLSv1.1 -TLSv1.2
+    SSLProtocol           all -SSLv3 -TLSv1 -TLSv1.1
     SSLHonorCipherOrder   off
     SSLCompression        off
-    SSLSessionTickets     off
-    SSLUseStapling        On
-    SSLStaplingCache      "shmcb:logs/ssl_stapling(32768)"
+    SSLSessionTickets     on
+    SSLUseStapling        off
```

Comment:
Should we note why it's off here? And point to instructions in the apache docs on how to turn it on?